index.html

```diff
@@ -0,0 +1,200 @@
+                                                  <h4 class="title"><a href="#">Celebrities News update</a></h4>
+                                                  <p class="description">Music is the universal language of mankind ,
+                                                            Music is the mediator between the spiritual and the sensual
+                                                            life.."</p>
+                                        </li>
+                                        <li class="flex">
+                                                  <i class="bi bi-diagram-3 flex"></i>
+                                                  <h4 class="title"><a href="#">Upcoming Events</a></h4>
+                                                  <p class="description">Try creating your own sounds using opening
+                                                            source license rather than just watching.
+                                                            Entertainment has been saving life since 1800.
+                                                            Start a project on Entertainment today! </p>
+                                        </li>
+                                        <li class="flex">
+                                                  <i class="bi bi-cone-striped flex"></i>
+                                                  <h4 class="title"><a href="#">Ongoing Life Concert Show - Hall</a>
+                                                  </h4>
+                                                  <p class="description">"Music already change the world, because it
+                                                            change people ,and
+                                                            the wise musician are those who play what they can master!
+                                                  </p>
+                                        </li>
+                                        <li class="flex">
+                                                  <i class="bi bi-boombox-fill flex"></i>
+                                                  <h4 class="title"><a href="#">HYMUSIC Press Release</a></h4>
+                                                  <p class="description">"The true beauty of music is that it connects
+                                                            people.
+                                                            It carries a message, and we, the musicians, are the
+                                                            messengers." </p>
+                                        </li>
+                              </ul>
+                              <a href="#" class="btn flex" role="button">OUR PROGRAMMES</a>
+                    </section>
+                    <section id="cart">
+                              <div id="speakers-about" class="flex-column">
+                                        <h3>Featured Speakers</h3>
```

Comment:
- [ ] Kindly fix the image
![Capturefiximage](https://user-images.githubusercontent.com/81444338/175716091-0bebf4ff-e8eb-4869-b4b5-640961ac3f06.PNG)